micro-rdk/src/common/credentials_storage.rs

```diff
@@ -88,32 +91,22 @@ impl WifiCredentials {
     }
 }
 
-#[derive(Clone, Debug, Default)]
-pub struct TlsCertificate {
-    pub(crate) certificate: Vec<u8>,
-    pub(crate) private_key: Vec<u8>,
-}
-
-impl From<CertificateResponse> for TlsCertificate {
-    fn from(resp: CertificateResponse) -> Self {
-        Self {
-            certificate: resp.tls_certificate.into_bytes(),
-            private_key: resp.tls_private_key.into_bytes(),
-        }
-    }
-}
-
-pub trait NetworkSettingsStorage {
+pub trait WifiCredentialStorage {
```

Comment:
This is restoring the old name, correct?

Reply:
correct